cf-rd/src/main/java/org/eclipse/californium/tools/resources/RDNodeResource.java

```diff
@@ -38,11 +38,15 @@
 	private static final Logger LOGGER = Logger.getLogger(RDNodeResource.class.getCanonicalName());
 	
 	/*
+	 * Use one timer for keeping track of all RDNodeResource's lifeTime expiry tasks.
+	 */
+	private static final Timer lifetimeTimer = new Timer();
```

Comment:
I guess the timer will at least be run as a daemon thread or else it will prevent the JVM from exiting during shutdown of the server.

However, I think we should replace `Timer` with the `ScheduledThreadPoolExecutor` which seems to be the recommended approach for Java >= 5. It would also allow us to get rid of the `TimerTask` subclass...